scripts/prep/aqm/exevs_aqm_prep.sh

```diff
@@ -85,24 +87,24 @@ done
 checkfile=${DCOMIN}/${VDATE}/airnow/daily_data_v2.dat
 if [ -s ${checkfile} ]; then
     if [ -s ${conf_dir}/Ascii2Nc_daily_obsAIRNOW.conf ]; then
-        run_metplus.py ${conf_dir}/Ascii2Nc_daily_obsAIRNOW.conf $PARMevs/metplus_config/machine.conf
+        run_metplus.py ${conf_dir}/Ascii2Nc_daily_obsAIRNOW.conf ${PARMevs}/metplus_config/machine.conf
         export err=$?; err_chk
         cat $DATA/logs/${model1}}/metplus_daily_ascii2nc.log*
         mv $DATA/logs/${model1}/metplus_daily_ascii2nc.log* $DATA/logs
-	if [ $SENDCOM = "YES" ]; then
-	  cp ${PREP_SAVE_DIR}/airnow_daily_$VDATE.nc ${COMOUT}/${STEP}/${COMPONENT}/${RUN}.${VDATE}/${MODELNAME}
+	if [ ${SENDCOM} = "YES" ]; then
+	    cp ${PREP_SAVE_DIR}/airnow_daily_${VDATE}.nc ${COMOUTproc}
 	fi
     else
-        echo "WARNING: can not find ${conf_dir}/Ascii2Nc_daily_obsAIRNOW.conf"
+        echo "Warning: can not find ${conf_dir}/Ascii2Nc_daily_obsAIRNOW.conf"
     fi
 else
-  if [ $SENDMAIL = "YES" ]; then
-    export subject="AIRNOW ASCII Daily Data Missing for EVS ${COMPONENT}"
-    echo "Warning: No AIRNOW ASCII data was available for valid date ${VDATE}" > mailmsg
-    echo "Missing file is ${checkfile}" >> mailmsg
-    echo "Job ID: $jobid" >> mailmsg
-    cat mailmsg | mail -s "$subject" $maillist 
-  fi
+    if [ ${SENDMAIL} = "YES" ]; then
+        export subject="AIRNOW ASCII Daily Data Missing for EVS ${COMPONENT}"
+        echo "Warning: No AIRNOW ASCII data was available for valid date ${VDATE}" > mailmsg
```

Comment:
@Ho-ChunHuang-NOAA Can you please capitalize this WARNING? This capitalization is an NCO requirement.